connect-examples/v2/php_payment/process-payment.php

```diff
@@ -59,7 +57,7 @@
   if ($response->isSuccess()) {
     echo json_encode($response->getResult());
   } else {
-    echo json_encode($response->getErrors());
+    echo json_encode(array('errors' => $response->getErrors()));
```

Comment:
another way is to do something like this:
`  $create_payment_request = new CreatePaymentRequest($token, Uuid::uuid4(), $money);

  $response = $payments_api->createPayment($create_payment_request);
  $create_payment_response = new CreatePaymentResponse();
  $create_payment_response->setErrors($response->getErrors());
  $create_payment_response->setPayment($response->getResult());

  echo json_encode($create_payment_response->jsonSerialize());`